xml/obs_ag_source_publish.xml

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.1"
+ xml:id="_source_publish">
+ <title>Source Publisher</title>
+ <para>The job of the source publish service is to publish all sources for 
```

Comment:
* `source-publishing service` would sound much better to me, grammar-wise.
* You can make the sentence shorter by replacing `The job of the source publish service is to publish ...` with `The source-publishing service publishes ...`, but even then it sounds a little redundant. ;/